exercises/raindrops/runtests.jl

```diff
@@ -0,0 +1,73 @@
+using Base.Test
+
+#include("raindrops.jl")
+include("example.jl")
```

Comment:
Needs to be `include("raindrops.jl")` again.

If you run `julia runtests.jl raindrops` in the main directory, it tests the example solution. That way you don't have to manually change it in the exercise-specific test file.